src/imageops/sample.rs

```diff
@@ -750,16 +750,24 @@ where
 /// Resize the supplied image to the specified dimensions.
 /// ```nwidth``` and ```nheight``` are the new dimensions.
 /// ```filter``` is the sampling filter to use.
-pub fn resize<I: GenericImageView>(
+pub fn resize<I, P, S>(
```

Comment:
These other generic arguments aren't needed now either, right?

Reply:
I guess not